src/index.ts

```diff
@@ -381,5 +381,6 @@ async function handleProgramArgs() {
 
 void handleProgramArgs().catch(error => {
   console.error(error);
+  console.error(`Event-replay process failed due to the aforementioned error`);
```

Comment:
You should only show this error when the command arg was `import-events`

Reply:
Alternatively, you can concat the operand to this string so we have an error that would work for all operations e.g.:
```js
`${args.operand} process failed`
```